cfme/tests/perf/ui/test_ui_infrastructure.py

```diff
@@ -174,20 +178,22 @@ def test_perf_ui_infra_resource_pools(ui_worker_pid, soft_assert):
     if 'resource_pools' in perf_tests['ui']['page_check']['infrastructure']:
         nav_limit = perf_tests['ui']['page_check']['infrastructure']['resource_pools']
 
-    pages.extend(analyze_page_stat(perf_click(ui_worker_pid, prod_tail, True, sel.force_navigate,
-        'infrastructure_resource_pools'), soft_assert))
+    pages.extend(analyze_page_stat(perf_click(ui_worker_pid, prod_tail, True, navigate_to,
+                                              ResourcePool, 'All'), soft_assert))
 
     resource_pools = set([])
     for page in paginator.pages():
         for title in sel.elements("//div[@id='quadicon']/../../../tr/td/a[contains(@href,"
-                "'resource_pool/show')]"):
+                                  "'resource_pool/show')]"):
```

Comment:
this was done by Pycharm. It follows pep8 but I agree code should look similarly everywhere. So, I'm changing this back to old style.